CMakeLists.txt

```diff
@@ -42,31 +60,34 @@ set(SENTRY_BACKEND ${SENTRY_DEFAULT_BACKEND} CACHE STRING
   "The sentry backend responsible for reporting crashes, can be either 'none', 'inproc', 'breakpad' or 'crashpad'.")
 
 if(SENTRY_BACKEND STREQUAL "crashpad")
-	set(BACKEND_CRASHPAD TRUE)
+	set(SENTRY_BACKEND_CRASHPAD TRUE)
 elseif(SENTRY_BACKEND STREQUAL "inproc")
-	set(BACKEND_INPROC TRUE)
+	set(SENTRY_BACKEND_INPROC TRUE)
 elseif(SENTRY_BACKEND STREQUAL "breakpad")
-	set(BACKEND_BREAKPAD TRUE)
+	set(SENTRY_BACKEND_BREAKPAD TRUE)
 elseif(SENTRY_BACKEND STREQUAL "none")
-	set(BACKEND_NONE TRUE)
+	set(SENTRY_BACKEND_NONE TRUE)
 else()
 	message(FATAL_ERROR "SENTRY_BACKEND must be one of 'crashpad', 'inproc', 'breakpad' or 'none'")
 endif()
 
-if(BACKEND_CRASHPAD AND NOT (APPLE OR WIN32))
+if(SENTRY_BACKEND_CRASHPAD AND NOT (APPLE OR WIN32))
 	message(FATAL_ERROR "The Crashpad backend is currently only supported on macOS and Windows")
 endif()
-if(BACKEND_BREAKPAD AND NOT LINUX)
+if(SENTRY_BACKEND_BREAKPAD AND NOT LINUX)
 	message(FATAL_ERROR "The Breakpad backend is currently only supported on Linux")
 endif()
-if(BACKEND_INPROC AND WIN32)
+if(SENTRY_BACKEND_INPROC AND WIN32)
 	message(FATAL_ERROR "The in-process backend is not supported on Windows")
 endif()
 
+message(STATUS "SENTRY_TRANSPORT=${SENTRY_TRANSPORT}")
+message(STATUS "SENTRY_BACKEND=${SENTRY_BACKEND}")
+
 if(ANDROID)
-	set(WITH_LIBUNWINDSTACK TRUE)
+	set(SENTRY_WITH_LIBUNWINDSTACK TRUE)
```

Comment:
This condition should be `SENTRY_BACKEND_INPROC AND ANDROID`, since we may add the breakpad backend soon aswell.